pkg/grpc/exec/exec.go

```diff
@@ -336,10 +336,9 @@ func GetProcessExit(event *MsgExitEventUnix) *tetragon.ProcessExit {
 		parent.RefDec()
 	}
 	if proc != nil {
-		tetragonEvent.Process = proc.GetProcessCopy()
+		// Use cached version of process
+		tetragonEvent.Process = proc.UnsafeGetProcess()
```

Comment:
is this really needed? the etragonEvent.Process is initialized with tetragonProcess earlier, which is basically proc.UnsafeGetProcess ... looks like we just need to call the proc.RefDec in here? like for the parent case

Reply:
makes sense ;-) !  done